code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -256,6 +294,142 @@
 	path = /obj/item/clothing/accessory/badge/idbadge/intel
 
 /datum/gear/accessory/sleeve_patch
-	display_name = "shoulder sleeve patch"
+	display_name = "shoulder sleeve patch (colourable)"
 	path = /obj/item/clothing/accessory/sleevepatch
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/namepin
+	display_name = "pin tag (colourable)"
+	path = /obj/item/clothing/accessory/badge/namepin
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/asym
+	display_name = "asymmetric jacket (selection)"
+	path = /obj/item/clothing/accessory/asymmetric
+
+/datum/gear/accessory/asym/New()
+	..()
+	var/asyms = list()
+	asyms["blue asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric
+	asyms["purple asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric/purple
+	asyms["green asymmetric jacket"] = /obj/item/clothing/accessory/asymmetric/green
+	gear_tweaks += new/datum/gear_tweak/path(asyms)
+
+/datum/gear/accessory/sash
+	display_name = "sash (colourable)"
+	path = /obj/item/clothing/accessory/sash
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/accessory/cowledvest
+	display_name = "cowled vest"
+	path = /obj/item/clothing/accessory/cowled_vest
+
+/datum/gear/accessory/halfcape
+	display_name = "half cape"
```

Comment:
No capes please. We removed cloaks a while ago due to their conflict with the station and the setting's aesthetics.

Reply:
Being the head of Tajaran lore, I trust you can enlighten me as to why this conflicts with the aesthetics.

Reply:
When it comes down to xeno loadout, they should be there to represent something in the lore/ethos of the species, which may grant them some liberty in regards to this stuff. Which is not exactly the case with regular clothing in the loadout. Like I pointed out in another comment, the discussion in regards to this already happened some time ago.

Reply:
I also wish to bring up Skrell, the first species that Humanity had come in contact with well before our current time, if my recollection is correct and currently up-to-date. They also wear capes made available to them in the loadout and while I also actually don't want every single human to be running around the station with capes, I still wish to understand the mindset behind this outside of a worst-case scenario which seems to only exist as a potential—a potential that you can generally trust the community has already learnt to avoid, that is.

Reply:
Looking at the discussion, I get the impression that the reason for the removal of cloaks and capes is based on the notion that having them in the loadout would indicate they're NT-branded.
Now, I do disagree and I do feel that having NT-branded cloaks (departmental cloaks) would be beneficial and CONTRIBUTE to their corporate image of uniform they're going for, and would help get more people to get behind it. You can still note a particular marketing strategy as with Dubai Airlines, but one should also consider how many people want to work there for what their reasons may or may not be, versus working at another airline which may be more accommodating of their preferences that they aren't willing to give up. I'm not a professional in this field so I don't want to act like I'm hot shit with this matter.
BUT, this was the decision made regardless so I will respect that and I will remove the departmental cloaks as was intended. However, I don't see how the half cape, full cape, and the colourable cloak have to concede to this decision considering they avoid the issue presented. Skull said he was fine with having cloaks and capes in the loadout, just not NT departmental ones, if I'm reading this correctly.

Reply:
I know this is also going 'what about' but what about departmental ponchos?

Reply:
I agree that departmental ponchos are also silly and should be removed, but this is not the pr for it. The decision on cloaks was already made by the people in command and there is little to discuss on git, bring it to the forums.